README.md

```diff
@@ -6,5 +6,18 @@ Fork and clone this repository.
 
 ### Questions
 1. Explain why Rails exists and what it's used for.
+
+Rails exists as a framework to combine Ruby with HTML, CSS, and JavaScript to create a web application.
```

Comment:
This isn't wrong, per se, but it doesn't answer the question well.

Rails is a framework for rapidly building web applications. It does this by providing developers with boilerplate code and with generators that can be used to make more code easily.